<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-DGE33NL" failures="0" tests="7" name="com.example.demotestdependency.TestGroupDependencyTest" time="0.028" errors="1" timestamp="2022-04-29T13:53:28 IST" skipped="4">
  <testcase classname="com.example.demotestdependency.TestGroupDependencyTest" name="test7" time="0.000">
    <skipped/>
  </testcase> <!-- test7 -->
  <system-out/>
  <testcase classname="com.example.demotestdependency.TestGroupDependencyTest" name="test1" time="0.021">
    <error message="/ by zero" type="java.lang.ArithmeticException">
      <![CDATA[java.lang.ArithmeticException: / by zero
at com.example.demotestdependency.TestGroupDependencyTest.test1(TestGroupDependencyTest.java:8)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:135)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:673)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:220)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:945)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:193)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:808)
at org.testng.TestRunner.run(TestRunner.java:603)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:429)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:423)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:383)
at org.testng.SuiteRunner.run(SuiteRunner.java:326)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1249)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
at org.testng.TestNG.runSuites(TestNG.java:1092)
at org.testng.TestNG.run(TestNG.java:1060)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- test1 -->
  <system-out/>
  <testcase classname="com.example.demotestdependency.TestGroupDependencyTest" name="test4" time="0.001">
    <skipped/>
  </testcase> <!-- test4 -->
  <system-out/>
  <testcase classname="com.example.demotestdependency.TestGroupDependencyTest" name="test2" time="0.004"/>
  <system-out/>
  <testcase classname="com.example.demotestdependency.TestGroupDependencyTest" name="test3" time="0.002"/>
  <system-out/>
  <testcase classname="com.example.demotestdependency.TestGroupDependencyTest" name="test5" time="0.000">
    <skipped/>
  </testcase> <!-- test5 -->
  <system-out/>
  <testcase classname="com.example.demotestdependency.TestGroupDependencyTest" name="test6" time="0.000">
    <skipped/>
  </testcase> <!-- test6 -->
  <system-out/>
</testsuite> <!-- com.example.demotestdependency.TestGroupDependencyTest -->
